services/scan/src/precinct_scanner_app.ts

```diff
@@ -1,6 +1,6 @@
 import { OkResponse, Scan } from '@votingworks/api';
-import * as streams from 'memory-streams';
-import { Buffer } from 'buffer';
```

Comment:
I'm assuming the reorg here is just due to linting?

Reply:
I used the VS Code "Organize Imports" command to sort them. I prefer to just have an automatically sorted list of imports, though I don't think we've really codified that and different packages do things a bit differently.